index.js

```diff
@@ -1,445 +0,0 @@
-      true,
-    )
-
-    const padlock = new Item(
-      "Padlock",
-      "Just a standard pad lock.",
-      true
-    )
-
-    //lookup table
-    let itemLookUp = {
-      'note': note,
-      'backpack': backpack,
-      'key': key,
-      'flashlight': flashlight,
-      'runningShoes': runningShoes,
-      'rug': rug,
-      'padlock': padlock,
-    };
-
-    /*----------------------------------Rooms---------------------------------------------*/
-    //room class constructor
-    class Room {
-      constructor(name, desc, inv) {
-        this.name = name;
-        this.desc = desc;
-        this.inv = inv;
-        this.locked = false;
-      }
-    }
-
-    //rooms class
-    const gate = new Room(
-      "Gate",
-      "You quickly come upon a gated community that looks to overrun and long abandoned. The gate is loosely held together by some rusted chains with a note that reads: “We have safety and supplies beyond the town through the tunnel, there is a key in one of the houses that will let get through the door at the end of the cul-de-sac and into the tunnel and make your way to Paradise Cove. Hurry, once the gates are open there is no way to close them and ‘they’ will get in.",
-      []
-    );
-    const outside = new Room(
-      "Outside",
-      "Once you open the chains and push through the large gates, you see what is left of what was a nice gated community, but is now run down, burnt out or worse. On the left side of the street is a burnt shell of a house next to a decrepit old house with boarded up windows and what looks like a working front door. On the right side of the street are two houses, one is boarded up and wrapped in barbed wire, while the other sits barely touched. At the end of the cul-de-sac stands a giant white mansion, with giant marble columns and red door. There is a weathered but usable backpack",
-      ['backpack']
-    );
-    const kitchen = new Room(
-      "kitchen",
-      "Is just off the main entrance to the south. Where in the main hall there is a grand stair-way that is blocked by the roof that has collapsed and there’re terracotta tiles broken and strewn about.  The kitchen is very large. There is wall to wall cabinets from floor to ceiling covering most of the wall space. The cabinets are white in color and there isn't a single drawer or door or a single door on any of the cabinets.The cabinets are on the east, north and west side walls  In the center there is a very large island containing the stainless steel stove and propane grill cook station.The west side wall contains an large archway opening to what appears to be a living room.The east side wall contains a small but solid wooden door that has been boarded up and the boards are unable to be removed.The north side wall contains only cabinets.High up on a shelf there is what appears to be a brand new pair of PF Flyers running shoes.They are, Guaranteed to make you run faster and jump higher!",
-      ['runningShoes, key']
```

Comment:
having the array be listed as 'runningShoes, key' is saying that this is just one string in your array. you would either want it to be ['runningShes', 'key'] or [runningShoes, key] depending if your passing the variable or the string in as your array items. 

also having runningShoes be camel cased will throw errors when trying to access them because you're cleaning up your user input to be set to all lowercase. and it'll be comparing lowercase to camel case and throw an error saying that the item doesnt exist